contrib/buildrefactor/tests/python/pants_test/contrib/buildrefactor/test_print_target_integration.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants_test.pants_run_integration_test import PantsRunIntegrationTest
+
+
+class PrintTargetIntegrationTest(PantsRunIntegrationTest):
+  """Test Peek goal functionality
+
+      $./pants test contrib/buildrefactor/tests/python/pants_test/contrib/buildrefactor:print_target_integration
+  """
+
```

Comment:
could you also add a negative case here with an invalid target to show what the behavior should be?

Reply:
Updated to include additional test `test_print_invalid_name`.

Reply:
Thanks. Also I think it's probably better to address 
https://github.com/pantsbuild/pants/pull/5175#discussion_r155311087
https://github.com/pantsbuild/pants/pull/5175#discussion_r155311790
here since @itsedvard 's patch probably depends on this, and there are a lot more comments to address on that patch.

Reply:
Responded to your comments on the Meta Relocate PR:
TaskError - https://github.com/pantsbuild/pants/pull/5175#discussion_r155393973
`buildozer.py` refactoring -
 https://github.com/pantsbuild/pants/pull/5175#discussion_r155392587

I also updated `test_print_target_integration.py` with a `test_multiple_targets()` test for the raised TaskError.

Reply:
Had the CI fail with: 
`ERROR:
/home/travis/build/pantsbuild/pants/contrib/buildrefactor/src/python/pants/contrib/buildrefactor/print_target.py Imports are incorrectly sorted.` 
So I changed the import order in `print_target.py` so TaskError comes before ConsoleTask, as in `buildozer.py` and in an example I referenced: https://github.com/pantsbuild/pants/blob/d30cca1e0ecb9cc0e1b7e2cd0ff6e7e077e62a52/src/python/pants/backend/graph_info/tasks/list_owners.py

Reply:
`./pants fmt contrib/buildrefactor/src/python/pants/contrib/buildrefactor::` should fix it for you

Reply:
Thanks for the fix!